packages/app/src/components/App/index.js

```diff
@@ -57,6 +73,8 @@ RainbowFirebaseApp.propTypes = {
     translations: PropTypes.object,
     /** Object with your application Redux reducers. */
     reducers: PropTypes.object,
+    /** A funtion to initialize the app. AppSpiner will be visible while this function is running. */
```

Comment:
you have a typo `funtion` --> `function`. Also, we should specify it must be an async function